integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testCases/withAPIM/VhostAPIMTestCase.java

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
Not sure why this appears as a new file. Changes I made with https://github.com/wso2/product-microgateway/pull/2022/commits/a251a078b427fd02d1c97dc6c98934c6259f4cf7 had errors. Corrected them with https://github.com/wso2/product-microgateway/pull/2022/commits/35579b195a2ed4a3f304bb229711f2e00efff713